tests/layer_tests/tensorflow_lite_tests/test_tfl_Unary.py

```diff
@@ -19,6 +19,8 @@
     {'op_name': 'ELU', 'op_func': tf.nn.elu},
     {'op_name': 'EXP', 'op_func': tf.math.exp, 'kwargs_to_prepare_input': 'positive'},
     {'op_name': 'FLOOR', 'op_func': tf.math.floor},
+    {'op_name': '', 'op_func': partial(tf.nn.gelu, approximate=True), 'output_name_suffix' : '/Tanh'},
+#    {'op_name': '', 'op_func': partial(tf.nn.gelu, approximate=False), 'output_name_suffix' : '/Erf'},
```

Comment:
Let's have both tests

Reply:
with `Erf`, TFLiteConverter fails with error: 

```
>         raise converter_error
E         tensorflow.lite.python.convert_phase.ConverterError: Could not translate MLIR to FlatBuffer.<unknown>:0: error: loc(fused["Erf:", "UnaryOperation/Erf/Erf"]): 'tf.Erf' op is neither a custom op nor a flex op
E         <unknown>:0: note: loc(fused["Erf:", "UnaryOperation/Erf/Erf"]): Error code: ERROR_NEEDS_FLEX_OPS
E         <unknown>:0: error: failed while converting: 'main': 
E         Some ops are not supported by the native TFLite runtime, you can enable TF kernels fallback using TF Select. See instructions: https://www.tensorflow.org/lite/guide/ops_select 
E         TF Select ops: Erf
E         Details:
E         	tf.Erf(tensor<2x10xf32>) -> (tensor<2x10xf32>) : {device = ""}
```

I think this is limitation of current test logic and converter.
Any solution for this?

Reply:
Excatly, @jane-intel looks there is no easy fix for this, maybe adding 'TODO' comment at this point?